osgar/tools/repare_log.py

```diff
@@ -0,0 +1,35 @@
+"""
+  Utility for repairing log files with missing header
+"""
+import sys
+import os
+from osgar.logger import LogWriter
+
+
+def fix_log_header(filename):
+
+    with open(filename, 'rb') as f:
+        data = f.read()
```

Comment:
What is the expected size of the file you want to read into the memory? I wouldn't do this for GB of data.

Reply:
You are right. At the moment it is limitation say for logs < 1GB.